app/screens/Garage/Garage.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
-import {View, Text} from 'react-native';
+import React, {useState} from 'react';
+import {View, Text, ScrollView} from 'react-native';
+import CarInProgress from '../../components/CarInProgress/CarInProgress';
+import {useSelector, useDispatch} from 'react-redux';
+import {REDUCERWORKSHOPS} from '../../constants';
 
 import styles from './styles';
 
-const Garage = ({cars}) => {
+const Garage = ({route, navigation}) => {
+  const {title} = route?.params || {};
+  const cars = useSelector(store => store.garage[REDUCERWORKSHOPS[title]]);
+  const dispatch = useDispatch();
+
+  const fixCar = idx => {
+    const client = cars.find((value, index) => idx === index);
+    dispatch({type: 'Home', payload: {client, index: idx, type: title}});
```

Comment:
normally you would have a separate folder with action creators inside the store folder and define those in there. This one could be called sendCarHome and it will return this object. You would call it like dispatch(sendCarHome(client, idx, title));